app/controllers/home_controller.rb

```diff
@@ -1,4 +1,5 @@
 class HomeController < ApplicationController
   def index
+    @heros_names = Hero.all.map(&:name)
```

Comment:
I think that we should come up with a better name than `name`. If you look at the character cheatsheet, this is really a `Klasa Postaci`. Also the fact that user can give a "name" to his character will be additional source of confusion. Consider use `:character_class`, `character_type`.

P.S A very funny gotcha here. We shouldn't use just `:class` or `:type` because these words are reserved or have a very special meaning due to rails conventions. And we would get into the problems quickly :)